pom.xml

```diff
@@ -33,9 +33,15 @@
     <servicedoc.port>80</servicedoc.port>
     <!-- oasp4j-sample should actually be restaurant -->
     <servicedoc.basePath>oasp4j-sample/services/rest</servicedoc.basePath>
-    <servicedocgen.version>1.0.0-beta-3</servicedocgen.version>    
+    <servicedocgen.version>1.0.0-beta-3</servicedocgen.version>
   </properties>
-
+  <dependencies>
+    <dependency>
+      <groupId>javax.interceptor</groupId>
+      <artifactId>javax.interceptor-api</artifactId>
```

Comment:
You shall not add such dependency to toplevel POM as this adds the dependency to ALL subprojects what is simply wrong. E.g. the maven archetype template should not have this dependency nor the planned api module. Please let us know what error occurs where when you have this dependency removed again.